spec/spec.json

```diff
@@ -260,7 +272,16 @@
 	"deprecation": {
 	    "description": "Indicates that this particular element is considered deprecated in the API. See the description for details",
 	    "fields": [
-		{ "name": "description", "type": "string", "required": false }
+		{ "name": "description", "type": "string", "required": false },
+		{ "name": "attributes", "type": "[attribute]", "default": "[]" }
+	    ]
+	},
+
+	"attribute": {
+	    "description": "Represents an additional attribute that is attached to one of the objects in apidoc. The main use case is to caprue additional metadata that doesn't necessarily define the API but aids in code generation. Examples would be hints for certain code generators about classes to extend, interfaces to implement, annotations to add, names to assign to certain methods, etc. The specific attributes will be applicable only in the context of the specific code generators usings them.",
+	    "fields": [
+		{ "name": "name", "type": "string" },
+		{ "name": "value", "type": "object" }
```

Comment:
Only see name and value. I believe we should also have a description, as per most other objects. For documentation at least, since JSON does not allow you to do comments.